MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,40 @@
+# Maintainers Guide
+
+## Responsibilities
+
+### Work on Current Issues and Create New Issues
+
+* Maintainers can choose any issues to work on and should make new issues when needed. Maintainers should add issues that they are working on to the the `In Progress` column of this [roadmap board](https://github.com/opensearch-project/opensearch-benchmark/projects/1). Any issue added to the `In Progress` column should be properly labeled. For more information on how to properly label issues and PRs, see the [labels](#labels) section of this document.
+
+### Triage Issues
+* Maintainers should meet biweekly to triage issues. This involves assessing new, current, and old issues and prioritize them based on this [roadmap board](https://github.com/opensearch-project/opensearch-benchmark/projects/1).
```

Comment:
and prioritizing them based on the [roadmap board](https://github.com/opensearch-project/opensearch-benchmark/projects/1).